### PR TITLE
[OD-82] Establish connection to database from backend

### PR DESCRIPTION
**Pull request by rlom721**
Updated the following files:
        modified:   package-lock.json
        new file:   package.json
        modified:   server/package-lock.json
        modified:   server/package.json
        new file:   server/src/config/dbConfig.ts
        new file:   server/src/db/passwordDb.ts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "pg": "^8.8.0"
+  }
+}
```
Comment (mudk01) [inline package.json:5]: could you pls delete this file

File: server/src/config/dbConfig.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const passwordDb = {
```
Comment (mudk01) [inline server/src/config/dbConfig.ts:5]: change name from passwordDb

File: server/src/config/dbConfig.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    host: process.env.DB_HOST,
+    port: parseInt(process.env.DB_PORT!),
+    database: process.env.DB_NAME,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASSWORD
```
Comment (mudk01) [inline server/src/config/dbConfig.ts:10]: could you add `ssl: true` after password

File: server/src/config/dbConfig.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+};
+
+export default passwordDb;
```
Comment (mudk01) [inline server/src/config/dbConfig.ts:13]: change here too

File: server/src/db/passwordDb.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import pg from 'pg';
+import passwordDb from '../config/dbConfig';
+
+const { Pool, types } = pg;
+
+const passwordDbConfig = passwordDb;
```
Comment (mudk01) [inline server/src/db/passwordDb.ts:6]: also change file name and everything w/password lol

**Review by mudk01 — CHANGES_REQUESTED** [review]

File: server/src/config/dbConfig.ts (hunk 2 above)
Comment (rlom721) [reply server/src/config/dbConfig.ts:5]: yes sir

**Review by mudk01 — APPROVED** [review]
LGTM!

**Review by KeatonWakefield001 — COMMENTED** [review]
passed all manual test

**Review by KeatonWakefield001 — APPROVED** [review]
aprroving

**Pull request merged**